[2021-01-29 15:17:51,513] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: udac_example_dag.Run_data_quality_checks 2019-01-12T01:00:00+00:00 [queued]>
[2021-01-29 15:17:51,519] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: udac_example_dag.Run_data_quality_checks 2019-01-12T01:00:00+00:00 [queued]>
[2021-01-29 15:17:51,519] {taskinstance.py:1017} INFO - 
--------------------------------------------------------------------------------
[2021-01-29 15:17:51,520] {taskinstance.py:1018} INFO - Starting attempt 1 of 1
[2021-01-29 15:17:51,520] {taskinstance.py:1019} INFO - 
--------------------------------------------------------------------------------
[2021-01-29 15:17:51,527] {taskinstance.py:1038} INFO - Executing <Task(DataQualityOperator): Run_data_quality_checks> on 2019-01-12T01:00:00+00:00
[2021-01-29 15:17:51,529] {standard_task_runner.py:51} INFO - Started process 45315 to run task
[2021-01-29 15:17:51,537] {standard_task_runner.py:75} INFO - Running: ['airflow', 'tasks', 'run', 'udac_example_dag', 'Run_data_quality_checks', '2019-01-12T01:00:00+00:00', '--job-id', '17', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/udac_example_dag.py', '--cfg-path', '/var/folders/ys/hb038j3904j77l4dkb4zk2ym0000gq/T/tmpyjyef5dw']
[2021-01-29 15:17:51,539] {standard_task_runner.py:76} INFO - Job 17: Subtask Run_data_quality_checks
[2021-01-29 15:17:51,575] {logging_mixin.py:103} INFO - Running <TaskInstance: udac_example_dag.Run_data_quality_checks 2019-01-12T01:00:00+00:00 [running]> on host 1.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.ip6.arpa
[2021-01-29 15:17:51,603] {taskinstance.py:1230} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=udacity
AIRFLOW_CTX_DAG_ID=udac_example_dag
AIRFLOW_CTX_TASK_ID=Run_data_quality_checks
AIRFLOW_CTX_EXECUTION_DATE=2019-01-12T01:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-01-12T01:00:00+00:00
[2021-01-29 15:17:51,603] {data_quality.py:22} INFO - DataQualityOperator not implemented yet
[2021-01-29 15:17:51,606] {taskinstance.py:1135} INFO - Marking task as SUCCESS. dag_id=udac_example_dag, task_id=Run_data_quality_checks, execution_date=20190112T010000, start_date=20210129T131751, end_date=20210129T131751
[2021-01-29 15:17:51,616] {taskinstance.py:1195} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-01-29 15:17:51,629] {local_task_job.py:118} INFO - Task exited with return code 0
